2024-04-24 15:35:39 ERROR Elements not present after 0.5 seconds on page: https://capital.com/zh/trade-stocks
2024-04-24 15:35:39 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x562794a10ce3 <unknown>
#1 0x5627946ff4a7 <unknown>
#2 0x56279474a31d <unknown>
#3 0x56279474a3d1 <unknown>
#4 0x56279478d734 <unknown>
#5 0x56279476c43d <unknown>
#6 0x56279478abe9 <unknown>
#7 0x56279476c1b3 <unknown>
#8 0x56279473d24a <unknown>
#9 0x56279473dc1e <unknown>
#10 0x5627949d500b <unknown>
#11 0x5627949d8f5b <unknown>
#12 0x5627949c1121 <unknown>
#13 0x5627949d9ac2 <unknown>
#14 0x5627949a5fdf <unknown>
#15 0x5627949ffe38 <unknown>
#16 0x562794a00010 <unknown>
#17 0x562794a0fe34 <unknown>
#18 0x7f8363494ac3 <unknown>

2024-04-24 15:35:45 ERROR Elements not present after 0.5 seconds on page: https://capital.com/zh/trade-stocks
2024-04-24 15:35:45 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x562794a10ce3 <unknown>
#1 0x5627946ff4a7 <unknown>
#2 0x56279474a31d <unknown>
#3 0x56279474a3d1 <unknown>
#4 0x56279478d734 <unknown>
#5 0x56279476c43d <unknown>
#6 0x56279478abe9 <unknown>
#7 0x56279476c1b3 <unknown>
#8 0x56279473d24a <unknown>
#9 0x56279473dc1e <unknown>
#10 0x5627949d500b <unknown>
#11 0x5627949d8f5b <unknown>
#12 0x5627949c1121 <unknown>
#13 0x5627949d9ac2 <unknown>
#14 0x5627949a5fdf <unknown>
#15 0x5627949ffe38 <unknown>
#16 0x562794a00010 <unknown>
#17 0x562794a0fe34 <unknown>
#18 0x7f8363494ac3 <unknown>

2024-04-24 15:35:48 ERROR Elements not present after 0.5 seconds on page: https://capital.com/zh/trade-stocks
2024-04-24 15:35:48 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x562794a10ce3 <unknown>
#1 0x5627946ff4a7 <unknown>
#2 0x56279474a31d <unknown>
#3 0x56279474a3d1 <unknown>
#4 0x56279478d734 <unknown>
#5 0x56279476c43d <unknown>
#6 0x56279478abe9 <unknown>
#7 0x56279476c1b3 <unknown>
#8 0x56279473d24a <unknown>
#9 0x56279473dc1e <unknown>
#10 0x5627949d500b <unknown>
#11 0x5627949d8f5b <unknown>
#12 0x5627949c1121 <unknown>
#13 0x5627949d9ac2 <unknown>
#14 0x5627949a5fdf <unknown>
#15 0x5627949ffe38 <unknown>
#16 0x562794a00010 <unknown>
#17 0x562794a0fe34 <unknown>
#18 0x7f8363494ac3 <unknown>

2024-04-24 15:35:52 ERROR Elements not present after 0.5 seconds on page: https://capital.com/zh/trade-stocks
2024-04-24 15:35:52 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 86, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 305, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x562794a10ce3 <unknown>
#1 0x5627946ff4a7 <unknown>
#2 0x56279474a31d <unknown>
#3 0x56279474a3d1 <unknown>
#4 0x56279478d734 <unknown>
#5 0x56279476c43d <unknown>
#6 0x56279478abe9 <unknown>
#7 0x56279476c1b3 <unknown>
#8 0x56279473d24a <unknown>
#9 0x56279473dc1e <unknown>
#10 0x5627949d500b <unknown>
#11 0x5627949d8f5b <unknown>
#12 0x5627949c1121 <unknown>
#13 0x5627949d9ac2 <unknown>
#14 0x5627949a5fdf <unknown>
#15 0x5627949ffe38 <unknown>
#16 0x562794a00010 <unknown>
#17 0x562794a0fe34 <unknown>
#18 0x7f8363494ac3 <unknown>